### PR TITLE
Title, stars and maps components rendering correctly. Render tests included.

### PR DESCRIPTION
**Pull request by MichaelDurfey**
Trello ticket! https://trello.com/c/4pHQCgWD/16-implement-basic-html-structure-and-basic-css

### CODE REVIEW

File: client/src/components/Stars.jsx
Comment (ZeMunchkin) [inline client/src/components/Stars.jsx:2]: I like that you made a component to render the stars. Very clean.

File: __tests__/__snapshots__/renderSpec.jsx.snap
Comment (mattman240) [inline __tests__/__snapshots__/renderSpec.jsx.snap:256]: This is a huge file I am a little unfamiliar with so I would suggest maybe have one other person look at it who might be able to see things I missed.

File: __tests__/renderSpec.jsx
Comment (mattman240) [inline __tests__/renderSpec.jsx:15]: Good change made it a useful test

**Review by mattman240 — APPROVED** [review]
This is another huge pr everything looks good to me but to be safe maybe wait for someone else to review!

File: __tests__/__snapshots__/renderSpec.jsx.snap
Comment (ZeMunchkin) [inline __tests__/__snapshots__/renderSpec.jsx.snap:1]: Is this document something you wrote to go with your tests? Or is it something that was generated by jest? I'm not sure if you meant to commit this because I'm not sure what it's for. If it's intentional, then ignore this comment. :)

**Review by ZeMunchkin — APPROVED** [review]
Looks good overall. See my comments. Assuming the renderSpec.jsx.snap is a document that you intended to commit, then this is ready to go. If not, just take that one out before you merge. :) My main feedback is to try to break down your PR into smaller chunks. This one had over 1800 lines added.

File: __tests__/__snapshots__/renderSpec.jsx.snap
Comment (MichaelDurfey) [reply __tests__/__snapshots__/renderSpec.jsx.snap:256]: It's basically a snapshot file that jest makes to use in it's testing and to compare rendered data to while using the snapshot testing methods. It advises to commit and include in code review on the docs so i'm including it here.. Made my PR huge though.

File: __tests__/__snapshots__/renderSpec.jsx.snap
Comment (MichaelDurfey) [reply __tests__/__snapshots__/renderSpec.jsx.snap:1]: It is something generated by jest! It's a snapshot that is used by their snapshot testing methods to make sure components are rendering correctly.

**Pull request merged**
